rpg.py

```diff
@@ -2,29 +2,42 @@
 from queue import PriorityQueue
 from typing import List, Tuple
 import random
+import time
+
+
+class Item:
```

Comment:
Better named as `Potion`

Reply:
I plan on making more items that are not potions, like status healers as I plan on creating status conditions